_rules/element-lang-valid-de46e4.md

```diff
@@ -27,20 +27,21 @@ acknowledgments:
 
 This rules applies to any HTML element that:
 
-- has a [node document](https://dom.spec.whatwg.org/#concept-node-document) with a [content type](https://dom.spec.whatwg.org/#concept-document-content-type) of `text/html`; and
-- is a [descendant](https://dom.spec.whatwg.org/#concept-tree-descendant) in the [flat tree](https://drafts.csswg.org/css-scoping/#flat-tree) of a `body` element; and
-- has a `lang` attribute that is not empty ("").
+- has a [node document][] with a [content type][] of `text/html`; and
+- is a [descendant][] in the [flat tree][] of a `body` element; and
+- has a `lang` attribute that is not empty (`""`).
 
```

Comment:
Please note that the TF suggested that elements without text content not be applicable. Definitely belongs in a different PR, but that does mean this PR doesn't fully address the feedback from the TF.

Reply:
Could that not be an additional condition within this list, added with this PR?

Reply:
I added this as an additional condition within this list.